src/audio_unit/delay.rs

```diff
@@ -21,16 +20,15 @@ pub struct Delay {
 }
 
 impl Delay {
-    pub const MIN_DELAY_MS: DelayMs = 0;
-    pub const MAX_DELAY_MS: DelayMs = 10000;
-
-    pub fn new(stream_config: &StreamConfig, delay_ms: DelayMs) -> Result<(Self, Sender<Message>)> {
-        Self::validate_delay(delay_ms)?;
-
+    pub fn new(
+        stream_config: &StreamConfig,
+        delay_ms: DelayMs,
+        max_delay_ms: DelayMs,
+    ) -> Result<(Self, Sender<Message>)> {
```

Comment:
Make the audio unit completely configurable. It's the effect's responsibility to set the max delay.